### PR TITLE
Fix: network switching/repitative rpc calls/health check rpc for L2 / 

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard: Add associated issues, tickets, docs URL here.

closes: 

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

- fix: Eliminated redundant calls from the home component by relocating get balances/getFsInfo/get_Saves to a dedicated component.
- add: Loading Fs Info & Save on certain interval from save page for ETH network only
- chore: Comment update for use gas watcher.
- fix: Open wrong network modal when initializeAccount returns response `wrongnetwork`
- fix: open switch network modal when active network changes
- refactor: gas service to use getGasData instead of getGasPrice as it's deprecated.
- fix: Extended RPC Url Health Check for L2 providers too.
- fix: Updated the config for L2 rpc urls as list instead of string to support health check.
- refactor: cleanup commented code
- fix: network switching with/without connecting to MM
- 
## Testing

Describe how to test your new feature/bug fix and if possible, a step by step guide on how to demo this.

### CODE REVIEW

**Comment by wsdt** [comment]
Lovely! Encountered some of these network errors this week as well! Will check, thx!

File: src/containers/save/Save.tsx [hunk 1]
```diff
@@ -74,6 +84,15 @@ const Save = () => {
     getMaxTransferValue()
   }, [])
 
+  useInterval(() => {
+    if (accountEnabled /*== MetaMask is connected*/) {
+      if (activeNetwork === NETWORK.ETHEREUM) {
```
Comment (wsdt) [inline src/containers/save/Save.tsx:89]: I know this is intentional, but why do we only do ETH here?

**Review by wsdt — APPROVED** [review]

File: src/containers/save/Save.tsx (hunk 1 above)
Comment (sk-enya) [reply src/containers/save/Save.tsx:89]: We are only supporting stakes on the Ethereum network, it's fine to remove that from the component itself but just added for extra check.

Comment (sk-enya) [reply src/containers/save/Save.tsx:89]: But anyways let me give one more look and cleaned it up

**Pull request merged**